src/main/java/sceaj/adyenmock/api/v1/PaymentController.java

```diff
@@ -0,0 +1,31 @@
+package sceaj.adyenmock.api.v1;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+import sceaj.adyenmock.api.v1.model.payment.PaymentRequest;
+import sceaj.adyenmock.api.v1.model.payment.PaymentResponse;
+import sceaj.adyenmock.payments.PaymentService;
+
+@Slf4j
+@RestController
+@RequestMapping(path = "/payments", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
I changed it to follow this pattern. After you pointed this out, I looked at the documentation and noticed your point since the endpoints are segregated by domain and subdomains on the left menu.